README.md

```diff
@@ -108,13 +108,15 @@ bin/pep-proxy.js
 
 Once the PEP Proxy is working, it can be used to enforce both authentication and authorization over the protected component (e.g. Orion Context Broker). In order to enforce both actions, the PEP Proxy has to be connected to an [Identity Manager](https://github.com/ging/fi-ware-idm) server and an [Access Manager](https://github.com/telefonicaid/fiware-keypass) server. Next sections will show some examples of both processes.
 
-Note that, in order for a request to be processed, it's mandatory that it contains the all of following headers:
+Note that, in order for a request to be validated through the Access Control, it's mandatory that it contains the all of following headers:
 * **x-auth-token**: should contain a valid user token, generated by the IDM.
 * **fiware-service**: should contain the name of a service created in the IDM and the user must have access to it.
 * **fiware-servicepath**: should contain a route to a subservice, begining with a slash '/' symbol.
 
 This headers are used by the IDM and Access Control systems to make the decisions about the request, so if any of them are missing, the request will not progress any further, and will be rejected with a 400 HTTP error code. For a detailed explanation of the returned errors, please refer to the [API Error codes](errorcodes.md).
 
+The proxy can also work in an authentication-only mode (usin the `config.access.disable` flag), in which case the `fiware-service` and `fiware-servicepath` headers can be marked as optional, or checked anyway for validity (controlled by the `config.authentication.checkHeaders` flag). Header checking cannot be disabled for PEP Proxies performing validation processes.
```

Comment:
Partly deferred to tech debt issue at https://github.com/telefonicaid/fiware-orion-pep/issues/177

NTC